packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.Props.ts

```diff
@@ -176,141 +172,4 @@ export interface IContextualMenuProps extends React.Props<ContextualMenu> {
    */
   calloutProps?: ICalloutProps;
 
-}
```

Comment:
I'm exporting the itemProps from the main props file so that if a user was pulling them directly from props, they'll still get them without interruption.